cypress/e2e/privatePaths/render/organizations/index.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { iPad2ForAppLayout, macbook15ForAppLayout } from "@utils/devices";
+import { newExpectationWithClickFlows, newClickFlow } from "@utils/helpers";
+import { runTestsForDevices } from "@utils/index";
+import {
+  shouldFindAndContain,
+  beVisibleAndContain,
+} from "@utils/sharedTests/sharedFunctionsAndVariables";
+
+const pageName = "Organization page: Visit Tabs";
+const currentPage = "/organizations/testorg";
+const loggedIn = true;
+
+const organizationPageFindAndContains = [
+  {
+    datacy: "create-deployment-button",
+    text: "Create Deployment",
+    clickToDataCy: "account",
+  },
+  {
+    datacy: "delete-organization-button",
+    text: "Delete Organization",
+    clickToDataCy: "members",
+  },
+  {
+    datacy: "add-member-button",
+    text: "Add member",
+    clickToDataCy: "teams",
+  },
+  {
+    datacy: "create-team-button",
+    text: "Create Team",
+    clickToDataCy: "billing",
+  },
+];
```

Comment:
I think this would be clearer if the format of this was:
```
{
  tabDataCy: "",
  dataCy: "",
  text: ""
},
```
And then you check the account tab first, and then iterate through each tab. Reads more clearly to have everything contained in a tab in one object